UPGRADE-6.2.md

```diff
@@ -24,3 +24,8 @@ Validator
 ---------
 
  * Deprecate the `loose` e-mail validation mode, use `html5` instead
+	 
+Serializer
+----------
+
+ * `AttributeMetadataInterface` has additional methods: `setSerializedPath()` and `getSerializedPath()`
```

Comment:
Add methods `setSerializedPath()` and `getSerializedPath()` to `AttributeMetadataInterface`